README.md

```diff
@@ -38,17 +38,17 @@ libraries.
 
 ```groovy
 dependencies {
-    // FirebaseUI for Firebase Realtime Database
-    compile 'com.firebaseui:firebase-ui-database:3.1.0'
-    
-    // FirebaseUI for Cloud Firestore
-    compile 'com.firebaseui:firebase-ui-firestore:3.1.0'
+// FirebaseUI for Firebase Realtime Database
+implementation 'com.firebaseui:firebase-ui-database:3.1.0'
```

Comment:
nit: can we preserve the indentation in the `dependencies` block? Here and in the instance below.

Reply:
Are you sure? Everywhere else, we 0 indent even though the real code would be indented. Otherwise, the indents get copied which can mess things up when pasting the code in an IDE.